src/components/eventCard/eventCardContainer.js

```diff
@@ -0,0 +1,39 @@
+import EventCard from "./eventCard.js";
+import Store from "../../redux/store";
+import React from "react";
+import "./eventCardContainer.css";
+
+export default function EventCardContainer(props) {
+  const [events, setEvents] = React.useState([]);
+  Store.subscribe(() => {
+    // console.log("hi");
+    let curr = Store.getState().newEvent.events;
```

Comment:
if this line is supposed to retrieve all events from redux, we should use useSelector